docs/installation.rst

```diff
@@ -99,7 +99,34 @@ Add django-debug-toolbar's URLs to your project's URLconf:
 This example uses the ``__debug__`` prefix, but you can use any prefix that
 doesn't clash with your application's URLs.
 
-5. Add the Middleware
+5. Configure django-debug-toolbar
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Add the configuration for django-debug-toolbar in your setting to run tests:
+
+.. code-block:: python
+
+    import sys
+
+    DEBUG = bool({"runserver"}.intersection(sys.argv))
```

Comment:
I don't like it. On my app the DEBUG is a bool fixed by the user. I can do bool even if I don't use the runserver.

Reply:
What suggestion do you have?

Reply:
@matthiask I borrowed this from a solution you provided [here](https://github.com/matthiask/workbench/blob/5e3663cb7c28cec707cfe746e9072e4fa6778f30/workbench/settings.py#L16-L17)

I am open to suggestions of how to make it better.

Reply:
> DEBUG = server_settings.get('DEBUG', False) if you extract the settings on a file (I do with a yaml) or DEBUG = os.environ.get("DEBUG", False) if you want to use an env variable

thanks let me test this out

Reply:
The previous solution was more like this 
`DEBUG = env("DEBUG", default=bool({"runserver"}.intersection(sys.argv)))`

I remove the env

Reply:
I think we can leave the DEBUG setting as-is in the instructions. The user can define how they want to configure that on their own.

Reply:
@VeldaKiara Yes, I have used that pattern for some time. I am now requiring that `DEBUG` is set in the environment in all new projects, it works better for me. It's maybe more annoying to start, but I like the fact that it's explicit instead of magic.